resources/local/config.toml

```diff
@@ -63,6 +63,13 @@ known_addresses = ['127.0.0.1:34553']
 # The interval (in milliseconds) between each fresh round of gossiping the node's public address.
 gossip_interval = 30000
 
+# Enable systemd support. If enabled, the node will notify systemd once it has synced and its
+# listening socket for incoming connections is open.
+#
+# It is usally better to leave this option off and enable it explicitly via command-line override
+# only in the unit files themselves via `-C=network.systemd_support=true`.
```

Comment:
@Fraser999 I am not entirely sure how it would behave on a non-systemd system, but I think it would simply fail, causing us to output a warning. It might delay for some time though trying to send the packet.

I'll try it out once I have the chance.